### ISSUE TITLE
AutoExplore config not working

### ISSUE THREAD

**Issue by Rimesrea** [InnohVateur/RandomMoons#2]
I set it to False but it still automatically explores and lands as soon as the ship goes to orbit.

![image](https://github.com/InnohVateur/RandomMoons/assets/164140051/c3c49a35-f9ea-45ae-937b-9e6b640fa3a7)

**Comment by B0operD0oper** [comment]
Having the same problem here. The mod explores moons automatically, regardless of what I put in the config.

**Comment by InnohVateur** [comment]
I'll be working on it this weekend, thanks. Probably due to CSync that I REALLY need to fix

**Comment by InnohVateur** [comment]
May I ask you if you were the host when you noticed the bug ?

**Comment by Rimesrea** [comment]
> May I ask you if you were the host when you noticed the bug ?

Yep. I was the host. Right now we're using v1.2.2 so it wouldn't auto land but we had a case when one player accidentally updated it to 1.2.3 and it also kept landing when he did the explore command even if all of us including me the host is on 1.2.2

**Comment by InnohVateur** [comment]
alr I'll work on it tomorrow thx

**Comment by InnohVateur** [comment]
So, it appeared I was busy with school. I'll try to do it this weekend but I still have a lot to do. Sorry ig... I'll keep you updated

**Comment by Dust29** [comment]
And he did not. If InnohVateur is okay with that (you are right innoh ?) I'm going to try to see if I can fix the issue. I'l keep you posted

**Comment by Dust29** [comment]
Okay so, would you mind trying to see if you can reproduce the bug with the 1.3.0 version ? we noticed a few bugs while testing, but I can't tell if it's the same as this one. 

Also, If the same bug is still happening, I have found a mistake in the code that might cause it, but I wan't to see if it's still happening before fixing it

Thanks for your time in advance
-Dust
